docs/websocket/feedback.md

```diff
@@ -123,6 +127,16 @@ id   | `string` | **Required** The id of the feedback to pause.
 
 A paused feedback can be resumed using this RPC. After this the audio stream is
 again processed by the server.
+Currently **it is required to re-send the audio header when resuming
+feedback**. Failing to do so will fail the feedback as the server can't
+recognise the audio format without a header.
+
+!!! note
+    It appears to be valid to have a wave file with multiple headers.
+
+!!! warning
+    It is required to re-send the audio header upon resume. The audio
+    processing will fail if the header isn't re-sent.
```

Comment:
Same comment as https://github.com/itslanguage/itslanguage-docs/pull/74#discussion_r181974515
It's not needed to have this redundant comment. It's already in bold above. Please remove one of the two.

Reply:
Done